### ISSUE TITLE
In examples/web, ./build.sh: could not find `graphql_client` in `{{root}}`

### ISSUE THREAD

**Issue by sgrove** [graphql-rust/graphql-client#238]
Thanks for updating the wasm example, it's really exciting to see!

I tried to compile it for a bit, and couldn't seem to make much headway though. Currently hitting this:

```
$ cd graphql-client/examples/web/ && ./build.sh
++ export CARGO_INCREMENTAL=0
++ CARGO_INCREMENTAL=0
++ cargo +nightly build --target wasm32-unknown-unknown --release
   Compiling web v0.1.0 (/private/tmp/graphql-client/examples/web)
error[E0433]: failed to resolve: could not find `graphql_client` in `{{root}}`
  --> src/lib.rs:11:10
   |
11 | #[derive(GraphQLQuery)]
   |          ^^^^^^^^^^^^ could not find `graphql_client` in `{{root}}`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: Could not compile `web`.

To learn more, run the command again with --verbose.

$ rustc --version
rustc 1.34.1 (fc50f328b 2019-04-24)
```

I'm not sure how to fix the missing `graphql_client` reference - any suggestions?

**Comment by tomhoule** [comment]
Thanks for the report! Indeed it's not building, here's a quickfix we can merge: https://github.com/graphql-rust/graphql-client/pull/239 - I think it broke somewhere with the change to the 2018 edition. We should build that example in CI.

It's not ideal though because graphql_client_web is already reexporting graphql_client, but the idea is to merge these crates soon and have the `web` part be a feature instead of a different crate.

**Comment by tomhoule** [comment]
This is fixed in 0.8.0 :)
